### ISSUE TITLE
.dmg defined as "damaged"

### ISSUE THREAD

**Issue by Sugy747** [Cubitect/cubiomes-viewer#361]
I downloaded and tried to run the .dmg on my Mac mini m4, but it doesn't work.
OS Version: macOS Sequoia 15.3.2.

From the attached image:
"cubiomes-viewer" is damaged and cannot be opened. You should eject the disk image.

<img width="932" alt="Image" src="https://github.com/user-attachments/assets/7397940a-ab1a-42be-89dc-4b7642b7ccab" />

**Comment by karbun01** [comment]
I think I found a solution to your problem. Check it out here [click](https://forreststonesolutions.com/robots/)

**Comment by hapiben** [comment]
Same here.

**Comment by EdricChan03** [comment]
From what I can tell, this is because the application isn't code-signed, as removing the `com.apple.quarantine` attribute that Apple adds on non-code-signed applications seems to resolve this warning dialog - you can use the `xattr` command to remove the flag:

```shell
xattr -r -d com.apple.quarantine <path/to/cubiomes-viewer.app>
```

However, the app still crashes - running the executable from the command-line shows this output:

```
❯ /Applications/cubiomes-viewer.app/Contents/MacOS/cubiomes-viewer
qt.qpa.plugin: Could not find the Qt platform plugin "cocoa" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

[1]    39774 abort      /Applications/cubiomes-viewer.app/Contents/MacOS/cubiomes-viewer
```

**Comment by sahbfhvhavckhjcvahkCVKhasdfa** [comment]
I got this too. I have an m1 MacBook air but I don't want to mess in terminal sorry, is there another way?

**Comment by miqumi** [comment]
sheesh
